lib/ansible/plugins/action/reboot.py

```diff
@@ -220,16 +228,39 @@ def get_system_boot_time(self, distribution):
 
         display.debug("{action}: getting boot time with command: '{command}'".format(action=self._task.action, command=boot_time_command))
         command_result = self._low_level_execute_command(boot_time_command, sudoable=self.DEFAULT_SUDOABLE)
+        stdout = command_result['stdout']
+        stderr = command_result['stderr']
 
         if command_result['rc'] != 0:
-            stdout = command_result['stdout']
-            stderr = command_result['stderr']
             raise AnsibleError("{action}: failed to get host boot time info, rc: {rc}, stdout: {out}, stderr: {err}".format(
                                action=self._task.action,
                                rc=command_result['rc'],
                                out=to_native(stdout),
                                err=to_native(stderr)))
-        display.debug("{action}: last boot time: {boot}".format(action=self._task.action, boot=command_result['stdout'].strip()))
+
+        filter_re = self._get_value_from_facts('BOOT_TIME_FILTER_REGEX', distribution, 'DEFAULT_BOOT_TIME_FILTER_REGEX')
```

Comment:
If we don't go with a default regex value, this can just be `None`.

```suggestion
        filter_re = self._get_value_from_facts('BOOT_TIME_FILTER_REGEX', distribution, None)
```